front-end/src/components/list/Mentees.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import ListGroup from "react-bootstrap/ListGroup";
+import Mentee from './Mentee';
+import ListGroupItem from 'react-bootstrap/ListGroupItem';
+import { Typeahead } from 'react-bootstrap-typeahead';
+import {sortAlphabetically} from '../../util/sorting';
+import { api } from '../../util/api';
+
+
+export default function Mentees(props) {
+    
+    function addMentee(mentee) {
+        if(mentee != null){
+            api('craftspeople/addmentee', {
+                method: 'PUT',
+                headers: {'Content-Type': 'application/json'},
+                body: JSON.stringify({
+                    'mentorId': props.craftsperson.id,
+                    'menteeId': mentee.id
+                })
+            });
+        }
+        props.rerender();
+    }
+
+    return (
+        <ListGroup data-testid="list">
+            {props.mentees.sort(sortAlphabetically).map(mentee => <Mentee key={mentee.id} rerender={props.rerender} mentee={mentee} />)}
```

Comment:
This line is very hard to read, we should put every `sort` `map` etc. on a new line + the use of <Mentee /> should be on a new line too